Gemfile

```diff
@@ -10,7 +10,9 @@ gem "rake"
 gem "sinatra"
 gem "sinatra-activerecord"
 gem "sinatra-flash"
+gem "erubi"
 gem 'securerandom'
+gem 'rack-protection'
```

Comment:
You didn't need to explicitly add this to the `Gemfile`, as it is already a dependency.